octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/util/error/ErrorComposite.java

```diff
@@ -86,90 +91,102 @@ public void widgetSelected(SelectionEvent e) {
+
+    private void displayGenericException(Exception ex) {
         compositeExceptionData.setLayout(new GridLayout(1, false));
-        
+
         TruncatingStyledText txtErrorField = new TruncatingStyledText(compositeExceptionData, SWT.MULTI);
         txtErrorField.setForeground(SWTResourceManager.getColor(SWT.COLOR_DARK_RED));
         txtErrorField.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, true, 1, 1));
-        
+
         txtErrorField.setText(ex.getMessage());
     }
-    
+
     private void displayOctaneException(OctaneException ex) {
         compositeExceptionData.setLayout(new GridLayout(2, false));
-        
+
         ErrorModel errorModel = ex.getError();
         String description = getDescriptionFromOctaneException(errorModel);
-        
-        if(description != null) {
-            TruncatingStyledText txtErrorField = new TruncatingStyledText(compositeExceptionData, SWT.NONE);
+
+        if (description != null) {
+            TruncatingStyledText txtErrorField = new TruncatingStyledText(compositeExceptionData, SWT.WRAP | SWT.MULTI | SWT.READ_ONLY);
             txtErrorField.setForeground(SWTResourceManager.getColor(SWT.COLOR_DARK_RED));
             txtErrorField.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false, 2, 1));
             txtErrorField.setText(description);
-            
+
             errorModel.removeValue(ERORR_MODEL_FIELD_DESCRIPTION);
             errorModel.removeValue(ERORR_MODEL_FIELD_DESCRIPTION_TRANSLATED);
         }
         
-        //Rest of values displayed generically
+        String fieldValueStackTrace = getFieldModelValueIfPresentAndNotEmpty(errorModel, ERROR_MODEL_FIELD_STACK_TRACE);
+        errorModel.removeValue(ERROR_MODEL_FIELD_STACK_TRACE);
+
+        // Rest of values displayed generically
         ex.getError().getValues().forEach(fieldModel -> {
-            
             String fieldValueTxt = getFieldModelValueIfPresentAndNotEmpty(errorModel, fieldModel.getName());
-            
-            if(fieldValueTxt == null) {
+            if (fieldValueTxt == null) {
                 return;
             }
-            
             Label lblErrorField = new Label(compositeExceptionData, SWT.NONE);
+            lblErrorField.setLayoutData(new GridData(SWT.BEGINNING, SWT.BEGINNING, false, false));
             lblErrorField.setText(convertFieldNameToLabel(fieldModel.getName()));
-            
+
             TruncatingStyledText txtErrorFieldValue = new TruncatingStyledText(compositeExceptionData, SWT.NONE);
-            txtErrorFieldValue.setLayoutData( new GridData(SWT.FILL, SWT.CENTER, true, false, 1, 1));
+            txtErrorFieldValue.setLeftMargin(6);
+            txtErrorFieldValue.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false, 1, 1));
             txtErrorFieldValue.setText(fieldValueTxt);
         });
+
+        Label lblStackTrace = new Label(compositeExceptionData, SWT.NONE);
```

Comment:
The stack trace might be empty, you still need an if to check if fieldValueStackTrace is empty

Reply:
Whoops